crates/bifrost/src/appender.rs

```diff
@@ -155,12 +165,16 @@ impl Appender {
     }
 
     #[instrument(level = "error" err, skip(retry_iter, bifrost_inner))]
-    async fn wait_next_unsealed_loglet(
+    async fn on_sealed_loglet(
         log_id: LogId,
         bifrost_inner: &Arc<BifrostInner>,
         sealed_segment: SegmentIndex,
         retry_iter: &mut RetryIter<'_>,
+        error_recovery_strategy: ErrorRecoveryStrategy,
     ) -> Result<LogletWrapper> {
+        // todo: configuration
+        let recovery_patience_dur = Duration::from_secs(5);
```

Comment:
The commit message says that it kicks in after 3s.

Reply:
It's not in configuration